Gems/Atom/Feature/Common/Code/Source/ImGui/ImGuiPass.cpp

```diff
@@ -653,7 +684,7 @@ namespace AZ
                 drawItem.m_indexBufferView = &m_indexBufferView;
                 drawItem.m_shaderResourceGroupCount = 1;
                 drawItem.m_shaderResourceGroups = shaderResourceGroups;
```

Comment:
These two lines can be removed if you switch to use PassSrg. 

And add this line in the beginning of this function:
` commandList->SetShaderResourceGroupForDraw(*m_resourceGroup->GetRHIShaderResourceGroup());`